src/ImGui.NET.FNA/ImVector.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace ImGuiNET.FNA
+{
+    public unsafe struct ImVector<T> where T : struct
```

Comment:
I don't think this structure is necessary, personally. I'm planning to improve the native ImVector struct with the auto-gen rewrite, so it will be usable in much the same way.  The way this is currently being used is pretty inefficient, anyways (reading/writing individual elements).